test-utils/src/lib.rs

```diff
@@ -17,6 +17,27 @@
 
 //! Test utils
 
+#[doc(hidden)]
+pub use futures;
+/// Marks async function to be executed by an async runtime and provide a `TaskExecutor`, suitable
+/// to test environment.
+///
+///	# Example
+///
+///	```
+///	use tokio; // WARNING: you must have tokio in the dependency of your crate to use this macro!
```

Comment:
```suggestion
```

This should be documented in the text above!

Reply:
done 0587d76

Reply:
fixed typo: e4b9ff1fc